src/lucky/renderable.cr

```diff
@@ -103,19 +103,66 @@ module Lucky::Renderable
     response.print
   end
 
+  private def handle_response(_response : Nil) forall T
+    # This is a hack to force Crystal to regenerate this method for
+    # each action. That way @type refers to the action with the problem,
+    # and not the first action to be compiled.
+    {% @type %}
+    {%
+      raise <<-ERROR
+
+
+      #{@type} returned Nil
+
+      But it should return a Lucky::Response.
+
+      Try this...
+
+        ▸ Ensure the action returns a response with html/redirect/json/etc.
```

Comment:
This felt like a path to me for a moment, what do you think about:
```suggestion
        ▸ Ensure the action returns a response with html, redirect, json, etc.
```

Reply:
Good idea!